jenkins/pipelines/cd/tiup/tiup-package-offline-mirror-dm.groovy

```diff
@@ -0,0 +1,70 @@
+if(DEBUG_MODE == "true"){
```

Comment:
The file is about the the motivation？

Reply:
the file is added to CI from jenkins job script